packages/devtron-fe-common-lib/package.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
skipping rest of the common-lib.

We can checkout: https://medium.com/@chris_72272/keeping-git-history-when-converting-multiple-repos-into-a-monorepo-97641744d928 or some similar stuff